hooks/pre-command

```diff
@@ -21,6 +21,12 @@ NVM_DIR=$(readlink -fn "$NVM_DIR")
 export NVM_DIR
 touch "$NVM_DIR/.automattic-nvm-dir-marker"
 
+# Pass the user specified .curlrc file content to all curl commands.
+if [[ -n ${BUILDKITE_PLUGIN_NVM_CURLRC:-} ]]; then
+    export CURL_HOME="$NVM_DIR"
+    echo "${BUILDKITE_PLUGIN_NVM_CURLRC}" > "$CURL_HOME/.curlrc"
```

Comment:
Doesn't that mean that this change in the curl config could bleed out? In other words, this parameter of the nvm plugin won't just affect `nvm install`'s underlying use of `curl` run as part of this `pre-command`, but could also affect any subsequent call to `curl` in the rest of that pipeline step (including any command run as part of the step's own `command:`)? 🤔

Reply:
Good catch! It should now be fixed in https://github.com/Automattic/nvm-buildkite-plugin/pull/15/commits/1738e7ed402af83d4ccdc7ffe8fb96805d242b64